recipe/meta.yaml

```diff
@@ -1,39 +1,49 @@
-{% set name = "uvloop" %}
 {% set version = "0.21.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: uvloop
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/u/uvloop/uvloop-{{ version }}.tar.gz
   sha256: 3bf12b0fda68447806a7ad847bfa591613177275d35b6724b1ee573faa3704e3
+  patches:
+    - 0000-always-cython.patch
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv --global-option='build_ext' --global-option='--cython-always' --global-option='--use-system-libuv'
+  number: 1
   skip: true  # [py<38 or win]
+  script:
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
Is it possible to skip the patch and use the option flags here? This seemed to work well

Patching can sometimes be finicky with conflicts and such

Reply:
94a1f7e